xconfess-backend/.env.sample

```
@@ -1,5 +1,52 @@
+# THROTTLE_TTL=900             # 15 min (seconds)
+# THROTTLE_LIMIT=100           # max requests per TTL
+
+# POST_RATE_LIMIT=5
+# POST_RATE_WINDOW=60
+# GET_RATE_LIMIT=30
+# GET_RATE_WINDOW=60
+
+# ──────────────────────── DLQ Retention ──────────────────
+# DLQ_RETENTION_DAYS=30
+# DLQ_CLEANUP_BATCH_SIZE=100
+# DLQ_DRY_RUN=true
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)